examples/openfin/platform.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
do we still need app.json?

Reply:
Actually it is nice being able to test both classic and platform

Reply:
Agreed, I think we want to be able to test both configs.